### PR TITLE
Added unit tests to welcome screen

### PR DESCRIPTION
**Pull request by emma3333**
- Added 3 x unit tests to welcome screen to test onPress event fires 
- Added 1 x snapshot test

### CODE REVIEW

**Comment by MikeD-ZOE** [comment]
Fixed the merge conflicts. The package-lock.json hopefully doesn't break things, we should remove it from the repor.

**Comment by JoshElgar** [comment]
> Fixed the merge conflicts. The package-lock.json hopefully doesn't break things, we should remove it from the repor.

I think Somesh thinks differently, he had a chat with people at Expo and they suggested leaving it in?

File: src/features/register/gb/tests/WelcomeScreen.test.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        const visitNhsText = 'visit the NHS website'
+        const onPressMock = jest.fn(() =>
+          Linking.openURL('https://www.nhs.uk/conditions/coronavirus-covid-19/')
+        )
+        const { getByTestId } = render(
+            <WelcomeScreen testID='nhsLink' onPress={onPressMock} />
+        )
+        fireEvent(getByTestId('nhsLink'), 'onPress', visitNhsText)
+        expect(onPressMock).toHaveBeenCalled()
+        expect(onPressMock).toHaveBeenCalledWith(visitNhsText)
```
Comment (JoshElgar) [inline src/features/register/gb/tests/WelcomeScreen.test.tsx:25]: Not sure if both lines are needed - the second line is probably enough.

**Comment by jamesbrooks94** [comment]
Package locks should be committed into repositories, whether that’s NPM or Yarn

https://github.com/npm/npm/blob/v5.0.0/doc/files/package-lock.json.md

**Comment by MikeD-ZOE** [comment]
I'm going to attempt to merge this in again.

**Pull request closed without merging**